package.json

```diff
@@ -46,7 +46,8 @@
     ],
     "globals": {
       "ts-jest": {
-        "skipBabel": true
+        "skipBabel": true,
+        "tsConfig": "<rootDir>/test/tsconfig.json"
```

Comment:
We are now telling ts-jest to compile using the `test/tsconfig.json` file just to be explicit.